numpy/core/tests/test_multiarray.py

```diff
@@ -5327,12 +5275,8 @@ def test_field_names(self):
             assert_equal(b[['f1', 'f3']][0].tolist(), (2, (1,)))
 
         # non-ascii unicode field indexing is well behaved
-        if not is_py3:
-            pytest.skip('non ascii unicode field indexing skipped; '
-                        'raises segfault on python 2.x')
```

Comment:
Scary stuff! A little alarmed there's no issue reference here.

Reply:
Is it still scary if it's going away?

I traced the origin of the code to ~9 years ago https://github.com/numpy/numpy/commit/99fc1281d0401b2284aeb69359c6b2ed739ade7d but it doesn't offer any explanation.